src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue10805.cs

```diff
@@ -0,0 +1,23 @@
+﻿#if TEST_FAILS_ON_CATALYST  //The Format property is not supported on the Mac platform; it always displays the default format.
```

Comment:
Do we have a related open issue? if exists, could you include the link to the issue here? And if not, could you create a new one?